### ISSUE TITLE
Publish Releases

### ISSUE THREAD

**Issue by TippyLion28** [BestOwl/MyPhone#11]
Hi, I am having trouble building this app. It would be great if you could publish a release :)

**Comment by BestOwl** [comment]
I will publish to Microsoft Store, ETA is mid-July.

**Comment by BestOwl** [comment]
Sorry for the late ETA, but I don't have time to work on this project right now, lots of coursework this mouth.

**Comment by TippyLion28** [comment]
Understandable, focus on your coursework :)

**Comment by Semro** [comment]
I was able to install the app from GitHub Actions [artifacts](https://github.com/BestOwl/MyPhone/actions/runs/2469376566#:~:text=non%2Dnullable%20type.-,Artifacts,-Produced%20during%20runtime)

Phone and PC were successfully paired, but pressing "Connect" didn't do anything.

**Comment by BestOwl** [comment]
> I was able to install the app from GitHub Actions [artifacts](https://github.com/BestOwl/MyPhone/actions/runs/2469376566#:~:text=non%2Dnullable%20type.-,Artifacts,-Produced%20during%20runtime)
> 
> Phone and PC were successfully paired, but pressing "Connect" didn't do anything.

A more stable version will be released next month, stay tuned!

**Comment by 4388450** [comment]
> I was able to install the app from GitHub Actions [artifacts](https://github.com/BestOwl/MyPhone/actions/runs/2469376566#:~:text=non%2Dnullable%20type.-,Artifacts,-Produced%20during%20runtime)
> 
> Phone and PC were successfully paired, but pressing "Connect" didn't do anything.

Just try make a phone call, you will see the popup on the right.

**Comment by BestOwl** [comment]
Hello everyone! Sorry for not being able to publish a release in July. But we are almost there. Here are some things that still need to be done before the release:
- [ ] Improve the reliability of the OBEX stack (the backbone of SMS and phonebook access features).
  - [ ] Bug fixes
  - [ ] Unit tests
- [ ] UI for SMS and phonebook.

I'm currently working on OBEX. Once the OBEX is stable, UI could be very easy to implement.

**Comment by Void48** [comment]
> Hello everyone! Sorry for not being able to publish a release in July. But we are almost there. Here are some things that still need to be done before the release:
> 
> * [x]  Improve the reliability of the OBEX stack (the backbone of SMS and phonebook access features).
>   
>   * [x]  Bug fixes
>   * [x]  Unit tests
> * [ ]  UI for SMS and phonebook.
> 
> I'm currently working on OBEX. Once the OBEX is stable, UI could be very easy to implement.

can't wait :)

**Comment by dvidebaek** [comment]
@BestOwl do we have a ETA? :-)

**Comment by BestOwl** [comment]
@dvidebaek Still need about 3 days to finish the SMS UI, but unfortunately I am very busy this week. So ETA is postpone to Sept 15.

**Comment by jianboy** [comment]
I had install the MyPhoneTestingCert_GithubAction.cer, But I can not install it.
```
PS E:\downloads\MyPhone_1.0.0.0_x86_x64_ARM64> .\Install.ps1
Found bundle: E:\downloads\MyPhone_1.0.0.0_x86_x64_ARM64\MyPhone_1.0.0.0_x86_x64_ARM64.msixbundle

Installing app...
Found dependency package(s):
E:\downloads\MyPhone_1.0.0.0_x86_x64_ARM64\Dependencies\x86\Microsoft.WindowsAppRuntime.1.1.msix
E:\downloads\MyPhone_1.0.0.0_x86_x64_ARM64\Dependencies\x64\Microsoft.WindowsAppRuntime.1.1.msix
Add-AppxPackage : Deployment failed with HRESULT: 0x800B0109, A certificate chain processed, but terminated in a root
certificate which is not trusted by the trust provider.

error 0x800B0109: The root certificate of the signature in the app package or bundle must be trusted.

NOTE: For additional information, look for [ActivityId] 55a82cbf-c6d7-0001-1905-bc55d7c6d801 in the Event Log or use
the command line Get-AppPackageLog -ActivityID 55a82cbf-c6d7-0001-1905-bc55d7c6d801

At E:\downloads\MyPhone_1.0.0.0_x86_x64_ARM64\Add-AppDevPackage.ps1:487 char:13
+             Add-AppxPackage -Path $DeveloperPackagePath.FullName -Dep ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (E:\downloads\My...RM64.msixbundle:String) [Add-AppxPackage], Exception
    + FullyQualifiedErrorId : DeploymentError,Microsoft.Windows.Appx.PackageManager.Commands.AddAppxPackageCommand

Error: Could not install the app.
Press Enter to continue...:

```

![image](https://user-images.githubusercontent.com/11532015/190262818-c23ae4c2-40c7-4b5d-a359-55694ec83a7a.png)

**Comment by BestOwl** [comment]
Where did you install the certificate? The certificate must be install in either store: LocalMachine\TrustedRootCertificateAuthorities or LocalMachine\TrustedPeople
(Note that you can delete the certificate after you installed the app.)

**Comment by BestOwl** [comment]
Sorry, I become very busy last month, so the release date has to be postponed again (I cannot give an ETA for now) 😢.

**Comment by Skywalker8510** [comment]
take your time I'm just commenting here so I get an email when there's an eta update

**Comment by RokeJulianLockhart** [comment]
@Skywalker8510, you don't need to comment to subscribe.

[Click here to subscribe.](https://github.com/BestOwl/MyPhone/issues/11#:~:text=or%20pull%20requests-,notifications,-Customize)

![image](https://user-images.githubusercontent.com/42837531/222746733-3943722f-964e-46d5-84e6-b90c047ae990.png)

![image](https://user-images.githubusercontent.com/42837531/222746861-e80949cb-d066-422e-b714-edad65c623ff.png)
